netbox/dcim/tables/devices.py

```diff
@@ -466,6 +466,14 @@ class PowerPortTable(ModularDeviceComponentTable, PathEndpointTable):
             'args': [Accessor('device_id')],
         }
     )
+    maximum_draw = tables.TemplateColumn(
+        template_code='''{% if record.maximum_draw %}{{ record.maximum_draw }} W{% else %}&mdash;{% endif %}''',
```

Comment:
We should be able to reference `value` here instead of `record.maximum_draw` explicitly.